ispcrt/tests/mock_tests/ispcrt_mock_main.cpp

```diff
@@ -931,6 +931,66 @@ TEST_F(MockTestWithModuleQueueKernel, TaskQueue_Sync_zeEventQueryKernelTimestamp
     ASSERT_EQ(sm_rt_error, ISPCRT_DEVICE_LOST);
 }
 
+TEST_F(MockTestWithModuleQueueKernel, FenceCreate) {
+    auto tq = m_task_queue;
+    auto future = tq.launch(m_kernel, 0);
+    ASSERT_EQ(sm_rt_error, ISPCRT_NO_ERROR);
+    ASSERT_TRUE(Config::checkCmdList({CmdListElem::KernelLaunch}));
+    auto fence = tq.submitWithFence();
```

Comment:
What will happen if someone uses 
```
tq.submitWithFence();
tq.sync();
````
I'm afraid there can be a L0 error about closing command list twice.